### PR TITLE
skip arm64 for fips test

### PR DESCRIPTION
**Pull request by lwan-wanglin**
skip arm64 for fips test, and temporarily skip for ppc64le and s390x too

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-gcp-ipi-ovn-ipsec-arm64-controlplane-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse /pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-ovn-ipsec-p1-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-gcp-ipi-ovn-ipsec-arm64-controlplane-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-ovn-ipsec-f28

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-gcp-ipi-ovn-ipsec-f28

**Comment by lwan-wanglin** [comment]
pick single arm64 cluster, single amd64 cluster and multi-arch(amd64 + arm64) cluster, all jobs fips-check-fips-or-die steps succeed.

File: ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh [hunk 1]
```diff
@@ -121,14 +121,16 @@ if [[ $ret -eq 0 ]]; then
...
+mapfile -t node_archs< <(oc get nodes -o jsonpath --template '{range .items[*]}{.status.nodeInfo.architecture}{"\n"}{end}' | sort -R | uniq -d)
+echo "The architectures included in the current cluster nodes are ${node_archs[*]}"
+### Skip ppc64le and s390x temporarily becasue the test images are not manifest list images now.
+NON_SUPPORTED_ARCHES=(arm64 ppc64le s390x)
+for arch in "${node_archs[@]}"; do
+  if [[ "${NON_SUPPORTED_ARCHES[*]}" =~ $arch ]]; then
+      echo "Skip this test since it doesn't support $arch."
+      exit 0
+  fi
+done
```
Comment (prb112) [inline ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh:133]: Hi ... Is there an issue tracking when the images will be manifest listed?

Comment (lwan-wanglin) [reply ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh:133]: waiting for @aleskandro back from vacation, we will have a discussion and task creation.

Comment (prb112) [reply ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh:133]: great - thank you

Comment (lwan-wanglin) [reply ci-operator/step-registry/fips-check/fips-or-die/fips-check-fips-or-die-commands.sh:133]: @prb112 FYI. here is the jira issue https://issues.redhat.com/browse/MIXEDARCH-307

**Pull request merged**